src/compiler/crystal/command.cr

```diff
@@ -71,9 +70,6 @@ class Crystal::Command
     when "play".starts_with?(command)
       options.shift
       playground
-    when "deps".starts_with?(command)
```

Comment:
It can display the message like `Use 'shard' directly: 'crystal deps' is removed` instead of removing in silence.

Reply:
Yes, I think we can leave a deprecation message at least for one or two versions. What do others think?

Reply:
I agree.

Reply:
Fixed! We can discuss the deprecation message, but since it's going to be there for one version or two it might not be worth the time.